ui/raidboss/data/07-dt/raid/r3s.ts

```diff
@@ -87,10 +87,12 @@ const triggerSet: TriggerSet<Data> = {
       name: {
         en: 'Barbarous Barrage Uptime Knockback',
         de: 'Brutalo-Bomben Uptime Rückstoß',
+        cn: '塔击退提示功能',
```

Comment:
```suggestion
        cn: '击退塔uptime打法击退提示时机调整功能',
```
This setting seems decide how long the `Outputs.knockback` called after tower appear.
It not alert to use knockback immunity, but alert this time and after you should be knockback.